specs/spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'minitest'
+require 'minitest/spec'
+require 'minitest/autorun'
+require 'minitest/reporters'
+
```

Comment:
Let's require the module here--`require_relative ../farbar`--to load all of our data classes.